packages/design-system/src/components/Table/TableCell.jsx

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableCell = ({
+  children,
+  className,
+  data,
+  stackedTitle,
+  stackedClassName,
+  type,
+  ...attributeOptions
+}) => {
+  const classes = classNames(
+    'ds-c-table__cell',
+    className,
+    type === 'numeric' ? 'ds-c-table__cell--numeric' : null
+  );
+
+  const stackedClasses = classNames(
+    'ds-c-table--stacked__col-header',
+    'ds-u-font-weight--bold',
+    stackedClassName
+  );
+
+  const renderStackedTitle = () => {
+    const isValidStackedTitle = stackedTitle && stackedTitle.length > 0;
+
+    return (
+      { isValidStackedTitle } && (
+        <span aria-hidden="true" className={stackedClasses}>
+          {stackedTitle}
+        </span>
+      )
+    );
+  };
+
+  return (
+    <td className={classes} role="cell" {...attributeOptions}>
```

Comment:
Is `type` ever passed onto the `td`?

Reply:
Not yet, it defaults to 'text', will add numeric column to pass type='numeric'